src/styles/prose.scss

```diff
@@ -11,6 +11,12 @@
 .prose {
   @apply text-gray-500;
 }
+.prose p{
```

Comment:
What if we have something else in prose and not only `p`. Should we write new rule for every tag in prose? You should apply this globally on line 12 in this file